src/main/java/ch/njol/skript/test/runner/EffObjectives.java

```diff
@@ -92,31 +92,24 @@ public String toString(@Nullable Event event, boolean debug) {
 	 * @return boolean true if the test passed.
 	 */
 	public static boolean isJUnitComplete() {
-		if (requirements.isEmpty())
-			return true;
-		if (completeness.isEmpty() && !requirements.isEmpty())
-			return false;
+		if (completeness.isEmpty() && requirements.isEmpty())
+			assert false;
```

Comment:
is this supposed to be an assert? if so:
```suggestion
		assert !completeness.isEmpty() || !requirements.isEmpty();
```